manifests/controller/controller-deployment.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  labels:
+   kubeless: controller
```

Comment:
Do we want to be consistent in the labels? Now I am using `controller=kubeless-controller` and `app=kafka, app=zookeeper`